pom.xml

```diff
@@ -16,7 +16,7 @@
 
 		<!-- DEPENDENCIES VERSION -->
 		<jdk.version>1.8</jdk.version>
-		<cds.services.version>1.15.3</cds.services.version>
+		<cds.services.version>1.16.0</cds.services.version>
 		<spring.boot.version>2.4.4</spring.boot.version>
```

Comment:
please also bump version of spring-boot to 2.4.5

Reply:
ok, will do